data/alpha/field/hostname.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: hostname
+phase: alpha
+register: ''
+text: The code representing a section of government service domain that identifies a service.
```

Comment:
Think @psd should review this field description text. Usually we try to have a generic description, to allow field reuse across registers. But we can merge for now.